source/team/roadmap.html.md.erb

```diff
@@ -1,10 +1,11 @@
 ---
 owner_slack: "#modernisation-platform"
 title: Our roadmap
-last_reviewed_on: 2023-11-21
+last_reviewed_on: 2024-02-21
 review_in: 3 months
 ---
 
 # <%= current_page.data.title %>
+This is our roadmap you can [view our roadmap here](https://app.productplan.com/pr/rY8sb6v1YYLZl4_dSKNSrUbasx2Ivzob).
 
-You can [view our roadmap here](https://app.productplan.com/pr/rY8sb6v1YYLZl4_dSKNSrUbasx2Ivzob).
+This is were we are mapping out the fture of the modernisation platform.
```

Comment:
This was probably fine as it was. It reads a little clunkily now; it mostly just repeats the same thing in different ways.